pdf/model/testdata/simple.txt

```diff
@@ -0,0 +1,12 @@
+1 0 obj
```

Comment:
Move this testdata to contrib/testdata subfolders?  e.g. contrib/testdata/font

Reply:
Is ok to move?

Reply:
Yes. I can do that when I am finished with the other changes